packages/bcer-retailer-app/app/src/utils/noi.util.tsx

```diff
@@ -68,9 +90,28 @@ export class NoiUtil {
     );
   }
 
+  static renderAction(handleDownload: Function) {
+    return function (l: BusinessLocation) {
+      return (
+        <Tooltip title="Download PDF" placement="top">
+          <IconButton
+            style={{
+              color: '#0053A5',
+            }}
+            onClick={() => {
+              handleDownload(l);
+            }}
```

Comment:
just a nitpick - we can pass the function to the onClick without needing the wrapping curly-braces, i.e:
`onClick={() => handleDownload(l)}`